sdk/script/management-sdk.ts

```diff
@@ -208,7 +195,17 @@ export class AccountManager {
             .then(() => null);
     }
 
-    public releasePackage(appName: string, deploymentName: string, fileOrPath: File | string, description: string, appVersion: string, isMandatory: boolean = false, uploadProgressCallback?: (progress: number) => void): Promise<void> {
+    public getDeploymentMetrics(appName: string, deploymentName: string): Promise<DeploymentMetrics> {
```

Comment:
Should we expose this? We never exposed this via the CLI, so I wonder if we need to bother making it available in the SDK. It's probably fine as is.